docs/DeveloperGuide.md

```diff
@@ -105,7 +105,8 @@ The `UI` component,
 
 ### Logic component
 
-**API** : [`Logic.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/logic/Logic.java)
+**
+API** : [`Logic.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/logic/Logic.java)
```

Comment:
Should this be on the same line?

Reply:
Thanks for the catch. Due to the line wrap at 120, I have changed it to:
```
**API**:
[`Logic.java`](https://github.com/AY2324S1-CS2103T-T133/tp/tree/master/src/main/java/seedu/address/logic/Logic.java)
```

etc.